filetests/isa/x86/bconst-binemit.clif

```diff
@@ -0,0 +1,8 @@
+test binemit
+target x86_64
+
+function %test_b64() -> b64 {
+ebb0:
+[-, %r10]   v0 = bconst.b64 true    ; bin: 41 ba 00000001
```

Comment:
Nice! Can you move this in binary64.clif, please? And add one using rax as the output register please?

Reply:
Should I rename `bconst-run.clif` as well?

Reply:
Yeah, or even feel free to create a subdirectory "run" if you want to (I do not mind either way).